test/apiv2/20-containers.at

```diff
@@ -545,6 +545,27 @@ if ! grep -q '201 Created' "${TMPD}/headers.txt"; then
     exit 1
 fi
 
+if root; then
+  podman run -dt --name=updateCtr alpine
+  curl -XPOST -o ${TMPD}/response1.txt --dump-header ${TMPD}/headers1.txt -H content-type:application/json http://$HOST:$PORT/v4.0.0/libpod/containers/updateCtr/update -d '{"Memory":{"Limit":500000}, "CPU":{"Shares":123}}'
```

Comment:
This is horrible, but it's not your fault - I missed reviewing some changes to this file, and bad stuff crept in. I need to spend some time to figure out how to fix it. Stay tuned.

Reply:
I'm working on a fix for this, stay tuned.

Reply:
Please do it this way:
```bash
if root; then
  podman run -dt --name=updateCtr alpine
  echo '{"Memory":{"Limit":500000}, "CPU":{"Shares":123}}' >${TMPD}/update.json
  t POST libpod/containers/updateCtr/update ${TMPD}/update.json 201

  # Verify
  echo '{ "AttachStdout":true,"Cmd":["cat","/sys/fs/cgroup/cpu.weight"]}' >${TMPD}/exec.json
  t POST containers/updateCtr/exec ${TMPD}/exec.json 201 .Id~[0-9a-f]\\{64\\}
  eid=$(jq -r '.Id' <<<"$output")
  # 002 is the byte length
  t POST exec/$eid/start 200 $'\001\0025'

  podman rm -f updateCtr
fi
```